apps/bfd-pipeline/bfd-pipeline-rda-bridge/pom.xml

```diff
@@ -21,6 +21,7 @@
 
         <bfd-shared-utils.version>1.0.0-SNAPSHOT</bfd-shared-utils.version>
         <bfd-pipeline-rda-grpcs.version>1.0.0-SNAPSHOT</bfd-pipeline-rda-grpcs.version>
+		<checkstyle.skip>false</checkstyle.skip>
```

Comment:
Be sure the whitespace is consistent–looks like spaces are preferred over tabs in this document.

Reply:
done